### DISCUSSION TITLE
Domain assigning issue

### DISCUSSION THREAD

**Discussion by yigithanyucedag** [vercel/community#6458] [Account Issue]
### Question

I am experiencing an issue where, despite having previously linked a domain to a project that has since been removed, I am unable to reuse the domain for a new project. The domain does not appear in the Domains page of my account, suggesting it's not actively linked to any project, yet when attempting to add it to a new project, I encounter an error stating the domain is already in use by my account.

**Comment by amyegan** [comment]
Hey @yigithanyucedag. Your team may recently have been updated to our new account structure. In that case your domain may be associated with your personal account instead of your new hobby team. You can learn more about the change and how to update your domain in https://github.com/vercel/community/discussions/6298

**Reply by yigithanyucedag** [reply]
The issue has been resolved. Perhaps it was related to server caching. Thank you for your answer.
